README.md

```diff
@@ -58,6 +58,14 @@ The cosign verification of Rancher Prime images can be done with:
 slsactl verify <prime_image>:<tag>
 ```
 
+Images built for upstream projects can be verified with(currently only Cluster API):
+
+```bash
+slsactl verify upstream --certIdentityWorkflow=<workflow_name> <prime_image>:<tag>
+```
```

Comment:
As discussed off-line, the idea behind `slsactl` is to abstract most (if not all) complexity when verifying images across the Rancher ecosystem.

I am not opposed to have a new flag which can be used to override identity information, however, by default all images within the ecosystem should be verifiable with:
`slsactl verify <IMAGE>`

That way users don't need to know anything about the pipeline (nor reach Customer Support) to validate images.